### PR TITLE
avoid breaking change in DFM

### PR DESCRIPTION
**Pull request by superyyrrzz**
@vwxyzh  @hellosnow @herohua @chenkennt @qinezh @ansyral

### CODE REVIEW

File: src/Microsoft.DocAsCode.Dfm/DocfxFlavoredMarked.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ public static DfmEngineBuilder CreateBuilder(string baseDir)
 
         public static string Markup(string src, string path = null, ImmutableDictionary<string, string> tokens = null)
         {
-            var engine = CreateBuilder(null).CreateDfmEngine(new DfmRenderer(tokens));
+            var engine = CreateBuilder(null).CreateDfmEngine(new DfmRenderer() {Tokens = tokens});
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Dfm/DocfxFlavoredMarked.cs]: add one space ahead and after?

File: src/Microsoft.DocAsCode.Dfm/DfmRenderer.cs [hunk 2]
```diff
@@ -13,15 +13,11 @@ namespace Microsoft.DocAsCode.Dfm
...
-        private readonly ImmutableDictionary<string, string> _tokens;
         private static readonly DocfxFlavoredIncHelper _inlineInclusionHelper = new DocfxFlavoredIncHelper();
         private static readonly DocfxFlavoredIncHelper _blockInclusionHelper = new DocfxFlavoredIncHelper();
         private static readonly DfmCodeExtractor _dfmCodeExtractor = new DfmCodeExtractor();
 
-        public DfmRenderer(ImmutableDictionary<string, string> tokens = null)
-        {
-            _tokens = tokens;
-        }
+        public ImmutableDictionary<string, string> Tokens;
```
Comment (herohua) [inline src/Microsoft.DocAsCode.Dfm/DfmRenderer.cs]: no getter and setter?

**Comment by hellosnow** [comment]
:shipit:

**Comment by herohua** [comment]
:panda_face:

**Comment by vwxyzh** [comment]
🚵

File: src/Microsoft.DocAsCode.Dfm/DfmRenderer.cs [hunk 3]
```diff
@@ -13,15 +13,11 @@ namespace Microsoft.DocAsCode.Dfm
...
-        private readonly ImmutableDictionary<string, string> _tokens;
         private static readonly DocfxFlavoredIncHelper _inlineInclusionHelper = new DocfxFlavoredIncHelper();
         private static readonly DocfxFlavoredIncHelper _blockInclusionHelper = new DocfxFlavoredIncHelper();
         private static readonly DfmCodeExtractor _dfmCodeExtractor = new DfmCodeExtractor();
 
-        public DfmRenderer(ImmutableDictionary<string, string> tokens = null)
-        {
-            _tokens = tokens;
-        }
+        public ImmutableDictionary<string, string> Tokens { get; set; };
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Dfm/DfmRenderer.cs]: fixed...

**Pull request merged**